setup.py

```diff
@@ -9,8 +9,10 @@
         "dataclasses",
         "geojson",
         "importlib_resources",
-        "lxml==3.6.0",
-        "numpy~=1.19.0",
+        "lxml==5.3.0",
```

Comment:
I just noticed there was a PR to pin this but I don't know the backstory,
https://github.com/NASA-IMPACT/hls-vi/pull/34

It looks like there (shockingly) is a wheel for `lxml==5.3.0` for `py36`, so maybe this is fine? I'm guessing the pin to an older version was to match the old version of the C library in our container. Maybe we can bump this to use the binary and then drop the `libxml2-dev` and `libxslt1-dev`?